gbbs/benchmarks/BatchDynamicConnectivity/EulerTourTree/ETTree.h

```diff
@@ -591,6 +591,38 @@ struct ETTree {
         return edge_sum;
         //return edge_sum.first ^ edge_sum.second;
     }
+    
+    template <class KY, class VL, class HH>
+    sequence<sequence<sequence<std::pair<uintE, uintE>>>> batch_sub_tree_sum(sequence<std::pair<uintE,uintE>> v_p,
+            pbbslib::sparse_table<KY, VL, HH> edge_index_table)
```

Comment:
Just to confirm, do you mean to pass these parameters by value? That creates a copy and since this is a vector of some sort, that looks quite expensive. Maybe you want to use `const&`? See, e.g., https://stackoverflow.com/a/2582942/2769271

Reply:
Yes, and I've modified that part. Thanks for this reminder!

Reply:
Did you push those commits onto this branch? (I didn't see more commits on this PR.)